include/vsg/io/Options.h

```diff
@@ -87,6 +87,22 @@ namespace vsg
         /// Coordinate convention to assume for specified lower case file formats extensions
         std::map<Path, CoordinateConvention> formatCoordinateConventions;
 
+        enum ColorSpace
+        {
+            sRGB,
+            linearRGB
+        };
+
+        /// Color space to use for vertex colors for scene graph
```

Comment:
Is this essentially the target colour space that users want to create their scene graphs so they match with the framebuffer colour spaces settings?

Reply:
It's not necessarily to match the framebuffer's image format (it's probably more common to have it as the opposite of the framebuffer's image format), but it *is* to control the colour space that users want their scenegraph to use internally. Users who are doing explicit manual conversions in their shaders or are being intentionally gamma-incorrect (e.g. to faithfully draw legacy content) will probably want sRGB vertex and material colours, whereas most users will want linear vertex and material colours and will have their shaders operate entirely with linear colours, and rely on the automatic conversions on store that come with using an sRGB image format for the framebuffer.